### ISSUE TITLE
ScheduleTypeLimits: "controlmode" failed to match any enum values E+ 8.9.0

### ISSUE THREAD

**Issue by jmarrec** [NREL/OpenStudio#3075]
OpenStudio-resources ems.osm test is failing in E+ 8.9.0.

```
Program Version,EnergyPlus, Version 8.9.0-40101eaafd, YMD=2018.03.26 10:48,
   ** Severe  ** <root>[ScheduleTypeLimits][ControlMode][unit_type] - "controlmode" - Failed to match against any enum values.
   **  Fatal  ** Errors occurred on processing input file. Preceding condition(s) cause termination.
   ...Summary of Errors that led to program termination:
   ..... Reference severe error count=1
   ..... Last severe error=<root>[ScheduleTypeLimits][ControlMode][unit_type] - "controlmode" - Failed to match against any enum values.
```

The problem is the ScheduleTypeLimits `Unit Type` = "ControlMode" isn't handled by the forward translator, while it doesn't exist in the `Energyplus+.idd`
https://github.com/NREL/OpenStudio-resources/blob/develop/model/simulationtests/ems.osm#L1592

```
OS:ScheduleTypeLimits,
  {b9a1bcce-5c25-4af2-805e-72ed9582b2bf}, !- Handle
  ControlMode,                            !- Name
  0,                                      !- Lower Limit Value
  1,                                      !- Upper Limit Value
  Discrete,                               !- Numeric Type
  ControlMode;                            !- Unit Type  <--------- PROBLEM HERE!
```

**Likely Fix:** add "controlmode" to this if statement in [ForwardTranslateScheduleTypeLimits.cpp](https://github.com/jmarrec/OpenStudio/blob/develop/openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateScheduleTypeLimits.cpp#L71:L76)

**Comment by jmarrec** [comment]
There is a discrepancy between openstudio's `ProposedEnergy+.idd` and EnergyPlus' `Energy+.idd`

OpenStudio's ProposedEnergy+.idd has these extra keys:
```
'ClothingInsulation',
 'ControlMode',
 'LinearPowerDensity',
 'MassFlowRate',
 'Pressure',
 'RotationsPerMinute',
 'SolarEnergy',
 'VolumetricFlowRate'
```

**Comment by jmarrec** [comment]
Fixed by #3077
